app/assets/stylesheets/components/_carousel.scss

```diff
@@ -222,6 +222,87 @@ $slide-text-desktop: 18px;
       }
     }
   }
+
+//--------------------------------------------------
+// class - impacts
+//--------------------------------------------------
+
+  &--impacts {
+    @include carousel;
+    overflow: hidden;
+    height: 100%;
+
+    .carousel {
+      &__slides-container {
+        width: 100%;
+      }
+
+      &__slide {
+        @include responsive-styles(flex-direction, column, column, row);
+        height: 100%;
+        width: 100%;
+        display: flex;
+        float: left;
+      }
+
+      &__indicator {
+        @include carousel-indicator($white);
+      }
+    }
+
+    .carousel-slide {
+      &__content {
+        @include responsive-styles(flex-direction, column, row, row);
+        @include responsive-styles(padding, $gutter-medium, $gutter-extra-large $gutter-medium,auto);
+        @include responsive-styles(align-items, center, flex-start, flex-start);
+
+        @include slide-font-size();
+        background-color: $primary-light;
+        border-radius: 25px;
+
+        display: flex;
+        flex-direction: column;
+        flex: 100%;
+
+        @include breakpoint($medium) {
+          order: -1;
+        }
+      }
+
+      &__icon {
+        @include responsive-styles(max-width, 50%, 33%, 20%);
+        @include responsive-styles(padding, 0px, 0px 25px, 0px 25px);
+        @include responsive-styles(margin, 50px 0px 66px, 0px, 0px);
```

Comment:
rem-calc

Reply:
Fixed. `rem-calc` used instead of straight `px` values
<img width="1239" alt="image" src="https://user-images.githubusercontent.com/56026099/122786623-683d7c80-d2ac-11eb-88c1-a4a858ae247a.png">

Reply:
`px` can be removed and is unnecessary for zero values